pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1430,6 +1430,7 @@ type PeerStatSpec struct {
 // +kubebuilder:printcolumn:name="SecureEndpoint",type=string,JSONPath=`.status.info.secureEndpoint`
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
 // +kubebuilder:subresource:status
+// +kubebuilder:subresource:scale:specpath=.spec.gateway.instances,statuspath=.status.replicas,selectorpath=.status.selector
```

Comment:
choosing `spec.gateway.instances` as `specpath` because we already control the number of instances of the RGW pod using `spec.gateway.instances`